src/styles/blocks/card.css

```diff
@@ -0,0 +1,122 @@
+/* #region Card */
+.card {
+  display: flex;
+  flex-direction: column;
+
+  box-sizing: border-box;
+  width: 200px;
+  padding: 16px;
+  margin: 20px auto;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__product {
+  margin: 16px 3px 0;
+  width: 160px;
+  height: 134px;
+}
+
+.card__description {
+  margin-top: 40px;
+
+  font-family: Roboto, sans-serif;
```

Comment:
you're using `font-family:` 7 times - isn't it just easier to apply it to entire body?